index.html

```diff
@@ -0,0 +1,219 @@
+          <button type="button" class="btn btn-next" onclick="changeScreen(event)">Next </i></button>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-4">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">3. Now click in the center of the right patch.</h3>
+            <div class="input-img-op">
+
+            </div>
+        </div>
+        <div class="badge-img">
+          <img src="./public/images/coloriright.png" alt="">
+        </div>
+        <div class="warnings">
+          <h5>Your selection should look like this. <span class="learn">Learn more <i
+                class="fas fa-angle-double-right"></i></span></h5>
+        </div>
+        <div class="buttons-div">
+          <button type="button" class="btn btn-back" onclick="changeScreen(event)"><i class="fa fa-angle-double-left"
+              aria-hidden="true"></i> Go Back</button>
+          <button type="button" class="btn btn-next" onclick="changeScreen(event)">Next </i></button>
+        </div>
+      </div>
+
+    </div>
+    <div class="screen container-5">
+      <div class="box-1">
+        <div class="upper-wrapper">
+          <h2 class="head-screens">4. Results</h3>
+            <div class="input-img-op">
+
+            </div>
+        </div>
+        <div class="result">
+          <h4>The RIGHT side is</h4>
+          <h2>15 % DARKER</h2>
+          <h4>than the LEFT side</h4>
+        </div>
+        <div class="badge-img">
+          <img src="./public/images/colorileft.png" style="margin-right:2vw;" alt="" srcset="">
+          <img src="./public/images/coloriright.png" alt="">
+        </div>
+        <div class="warnings">
+          <h5>Your data is not stored online. Take a screenshot to save. <span class="learn">Learn more <i
+                class="fas fa-angle-double-right"></i></span></h5>
+        </div>
+        <div class="buttons-div">
+          <button type="button" class="btn btn-back" onclick="changeScreen(event)"><i class="fa fa-angle-double-left"
+              aria-hidden="true"></i> Go Back</button>
+          <button type="button" class="btn btn-over" onclick="changeScreen(event)">Start Over</button>
+        </div>
+      </div>
+
+    </div>
+  </div>.
+  <script>
+    function changeScreen(event) {
+      var op = event.target.className;
+      if (op.includes("btn-next")) {
+        $(event.target).parents(".screen").fadeOut("slow");
+        $(event.target).parents(".screen").next().fadeIn("slow");
+      } else if (op.includes("btn-back")) {
+        $(event.target).parents(".screen").fadeOut("slow");
+        $(event.target).parents(".screen").prev().fadeIn("slow");
+      } else if (op.includes("btn-over")) {
+        $(event.target).parents(".screen").fadeOut("slow");
+        $(".container-2").fadeIn("slow");
+      }
+    }
+
+    let dropArea = document.getElementById('drop-area');
+    ;['dragenter', 'dragover', 'dragleave', 'drop'].forEach(
```

Comment:
Is there an extra semicolon here?

Reply:
Yes, actually for the array elements to parse correctly.